dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -106,19 +105,30 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 
 			<main>
 				{front.pressedPage.collections.map((collection, index) => {
-					// TODO: We also need to support backfill and treats containers
-					const trails = collection.curated;
+					// TODO: We also need to support treats containers
+					// Backfills should be added to the end of any curated content
+					const trails = collection.curated.concat(
+						collection.backfill,
+					);
 					// There are some containers that have zero trails. We don't want to render these
 					if (trails.length === 0) return null;
+
+					const containerElement = renderContainer({
```

Comment:
Can we drop the function pattern here and just make this a component? I think this special pattern only exists in *one* case, `renderElement` which was done as a sort of subjective / for reasons change but ultimately if your function is a react component then it should _be_ a react component and follow the same rules (Pascal case, be rendered using `<Thing />` syntax, etc)

Reply:
Made this change, definitely felt like it cleaned up the code a lot, thanks!